packages/material-ui/src/styles/createMuiTheme.test.js

```diff
@@ -5,16 +5,16 @@ import { deepOrange, green } from '../colors';
 describe('createMuiTheme', () => {
   it('should have a palette', () => {
     const muiTheme = createMuiTheme();
-    assert.strictEqual(typeof createMuiTheme, 'function', 'should be a function');
-    assert.ok(muiTheme.palette, 'should have a palette');
+    assert.strictEqual(typeof createMuiTheme, 'function');
+    assert.strictEqual(typeof muiTheme.palette, 'object');
   });
```

Comment:
That is exactly the reason you use named matchers. Adding your own implementation reduces readability and can introduce bugs which you did. `typeof null === 'object'` while `assert.ok` would've caught this.

Reply:
We could use `assert.strictEqual(muiTheme.palette != null, true);` but I'm more interested in knowing the *palette* is an object instead of a boolean / string.